packages/widget_toolkit_pin/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## [0.3.0]
 * Added `savePinCodeInSecureStorage` to `PinCodeService` for saving the pin code in secure storage
 * Added `autoPromptBiometric` parameter to `PinCodeKeyboard` to automatically authenticate with biometrics if available
+* Removed `isPinCodeInSecureStorage` method from `PinCodeService`. Now this flag is set automatically when the pin code is saved in secure storage and accessed through the `getPinCode` method
```

Comment:
Would you consider `isPinCodeInSecureStorage` and `savePinCodeInSecureStorage` to be breaking changes?
Since these changes will require additional work from the developer that is using the package

Reply:
Absolutely, that's why we increase the version from 0.2.2 to 0.3.0

Reply:
In that case, we could add a separation between breaking and non-breaking changes, similar to what we did in version 0.1.0
```
[0.1.0]

    Improvements to error handling
    Errors of type ErrorModel thrown by PinCodeService are now caught by the PinCodeKeyboard.onError callback

Breaking changes:

    Update PinCodeService.verifyPinCode to return a Future<dynamic> instead of a bool
    Update PinCodeKeyboard.onAuthenticated signature to accept a dynamic parameter passed from PinCodeService.verifyPinCode on authentication success
```

Reply:
Done